### PR TITLE
fix(menu): Review menu spacings - FRONT-2156

### PR DESCRIPTION
**Pull request by papegaill**
- EC: https://ecl-preview-2064--europa-component-library.netlify.app/playground/ec/?path=/story/components-navigation-menu--default
- EU: https://ecl-preview-2064--europa-component-library.netlify.app/playground/eu/?path=/story/components-navigation-menu--default

### CODE REVIEW

File: src/implementations/vanilla/components/menu/_menu.scss [hunk 1]
```diff
@@ -138,14 +138,6 @@ $_mobile-item-hover-background: null !default;
   }
 }
 
-/* stylelint-disable-next-line order/order */
-@include breakpoints.up('m') {
-  .ecl-menu {
```
Comment (emeryro) [inline src/implementations/vanilla/components/menu/_menu.scss:143]: it seems that this spacing change is still needed, at least on EU:
image
On EC, it seems to be the case too, even if that's not very clear...
image

**Review by emeryro — CHANGES_REQUESTED** [review]
The menu items are fine now, thanks.

File: src/implementations/vanilla/components/menu/_menu.scss (hunk 1 above)
Comment (papegaill) [reply src/implementations/vanilla/components/menu/_menu.scss:143]: Thks for review @emeryro, this was already validated by Erika in the comment of the ticket. Specs not being clear during implementation. After checking, Erika also found that the spacing for tablet was much too large, especially without a site name. Do you want to have this checked by the designers one last time?

Comment (emeryro) [reply src/implementations/vanilla/components/menu/_menu.scss:143]: If you say that they validated, it is fine for me

**Pull request merged**